google/gax/__init__.py

```diff
@@ -111,21 +109,36 @@ def merge(self, options):
 
 
 class CallOptions(object):
-    """Encapsulates the overridable settings for a particular API call"""
+    """Encapsulates the overridable settings for a particular API call.
+
+    ``CallOptions`` are an optional arg for all GAX API calls.  They are used to
```

Comment:
Singular vs plural ``CallOptions``? You use "its" below on line 117, which I prefer, so maybe

  ``CallOptions`` *is* an.... *It is* used...